pyproject.toml

```
@@ -32,8 +32,8 @@ orjson = "^3.9.7, !=3.9.10" # TODO: remove inequality once dep resolution issue
 packaging = ">=21.3,<25.0"
 tenacity = "^8.2.3"
 # optional dependencies
-snowflake-sqlalchemy = { version = "<1.6", optional = true } # TODO: remove pin once we have tested with sqlalchemy 2.0 & snowflake
-sqlalchemy = { version = "<2.0", optional = true }  # TODO: see above ^
+snowflake-sqlalchemy = { version = ">=1.6", optional = true }
+sqlalchemy = { version = ">=2.0", optional = true }
```

```suggestion
sqlalchemy = { version = ">=2.0"}
```
Removing the `optional = true` makes sqlalchemy required.
It wouldn't make a difference for our docker container but it would for anyone who installs the agent directly via `pypi`.